media-video/obs-studio/metadata.xml

```diff
@@ -12,6 +12,7 @@
     internally.
```

Comment:
`<maintainer type="person">`
This is remind me, at g.o address without commit access is one of the reason we got these new `proxied="yes"` tags. I could make the change (on merge, don't worry about this) and add myself as proxy if @Chiitoo acks it so these PRs don't get lost.

I don't use obs (last time I streamed I used standalone ffmpeg :eyes:) but I still want it to see timely updates.

Reply:
I would not mind that, although I do wonder a bit if there might be someone better and willing to take care of the package (or all of the packages I maintain) altogether.

Reply:
I think you've been doing great with this personally, and hope you keep at it.

Given haven't heard back from @ngg after last short review, and this PR has waited and for a long time already, I'll go ahead and do the last few checks / fixes (just warning ahead of time so hopefully no work duplication).

I need to mask USE=browser on ppc64/x86 too given it's using amd64 binaries. Haven't checked if there's binaries for the rest but I have no real way to test this and I'll leave it how it is.

Reply:
Hi! Sorry for not responding, I've been busier than usually. I'll have time to make changed on the weekend if that's ok.

Reply:
Oh don't worry about it, unless you really want to do it yourself I'm happy to just do it (especially if you're busy).

You've done a lot here already.

Reply:
I could ask another question though, are you interested in co-maintaining this officially?